src/popup/pages/Send/index.js

```diff
@@ -149,9 +149,16 @@ class SendPage extends React.Component {
                 })
                 .map(acc => acc.address)
 
+              const ledgerAddresses = this.state.allAccounts.commonList
+                .filter(acc => acc.type === ACCOUNT_TYPE.WALLET_LEDGER)
+                .map(acc => acc.address)
+
               if (!ownAddresses.includes(this.state.toAddress)) {
                 return getLanguage("confirmDepositingToParatimeToForeignAccount", "Destination account is not in your wallet! We recommend you always deposit into your own ParaTime account, then transfer from there.")
               }
+              if (ledgerAddresses.includes(this.state.toAddress)) {
+                return getLanguage("confirmDepositingToParatimeToLedgerAccount", "Destination account was imported from Ledger! Ledger accounts do not support withdrawing from ParaTime.")
```

Comment:
this being the non-evm side of the branch, it may be more comprehensive to say that Ledger accounts don't support ParaTimes at all? or am I mistaken about that?

the possible risk here is if people think "I'll transfer from Ledger to a non-Ledger account when I want to withdraw" and then find out they can't do that either

Reply:
Ah, yeah. Not sure of what the best text would be
- `Ledger accounts do not support withdrawing from ParaTime.` misses some operations
- `Ledger accounts do not support ParaTimes.` I think doesn't sound dangerous enough

Reply:
Deferred into https://github.com/oasisprotocol/oasis-wallet-ext/issues/238